statsmodels/imputation/mice.py

```diff
@@ -767,7 +767,7 @@ def plot_fit_obs(self, col_name, lowess_args=None,
             containing x-axis jitter and y-axis jitter, respectively.
         plot_points : bool
             If True, the data points are plotted.
-        ax : matplotlib axes object
+        ax : AxesSubplot
```

Comment:
why do you remove `matplotlib` everywhere?
AFAIK we only support matplotlib

Reply:
It is not needed.  It also renders poorly.

Reply:
Types like Series, Figure, ndarray, DataFrame, Axes, AxesSubPlot are all distinctive enough to need to further qualification.